src/ows/src/main/java/org/geoserver/ows/Request.java

```diff
@@ -21,7 +21,7 @@
  * {@link Dispatcher} while doing its dispatching work. In case of dispatching
  * exceptions some fields may be left blank, depending how far the dispatching
  * went.
- * 
+ *
  * @author Justin DeOliveira
```

Comment:
Not sure if all the reformat change sin this file were intentional.

Reply:
It was not, I'll remove this change.